### ISSUE TITLE
Build does not find errors when targeting ES5 with latest TypeScript version

### ISSUE THREAD

**Issue by HermannGruber** [TypeStrong/ts-loader#941]
### Expected Behaviour
If I modify the tsconfig.json in the "vanilla" example to target ES5, I expect a compiler error during `yarn build` after adding a statement using array.prototypes.includes, which is not part of the ES5 spec:

`TS2339: Property 'includes' does not exist on type '"foobar"'.`

I expect that it behaves in the same way if I use the latest version of TypeScript.

### Actual Behaviour
If I modify the package.json to use TypeScript version 3.2.1 (or later), `yarn build` emits javascript that contains the "includes" statement (non-ES5-compatible), and finishes successfully.

### Steps to Reproduce the Problem

Open the folder examples\vanilla in the ts-loader repo.

Add target es5 to tsconfig.json
`		"target": "es5"`

Modify Typescript version in package.json to use TypeScript 3.2.1:
`    "typescript": "3.2.1",`

Add the following line of code at the end of the `render()` method in render.ts:
`    var foo : boolean = "foobar".includes(":"); `

Finally, run `yarn install` and then `yarn build`. The latter gives the following output:

yarn run v1.16.0
warning package.json: No license field
$ webpack --mode production
Hash: 3a94c4e753e17ee5e58b
Version: webpack 4.0.0
Time: 2641ms
Built at: 2019-5-28 10:57:23
  Asset      Size  Chunks             Chunk Names
main.js  5.47 KiB       0  [emitted]  main
   [0] ./src/render.ts 182 bytes {0} [built]
   [1] ./src/index.ts 125 bytes {0} [built]
Done in 4.04s.

After changing the TypeScript version to 3.1.1 in package.json... 

`    "typescript": "3.1.1",`

...yarn install and yarn build yields the following output (as expected):

yarn run v1.16.0
warning package.json: No license field
$ webpack --mode production
Hash: 18ef7b411e508dced150
Version: webpack 4.0.0
Time: 2739ms
Built at: 2019-5-28 10:54:22
 1 asset
   [0] ./src/render.ts 182 bytes {0} [built] [1 error]
   [1] ./src/index.ts 125 bytes {0} [built]

ERROR in C:\1_Projects\ts-loader\examples\vanilla\src\render.ts
./src/render.ts
[tsl] ERROR in C:\1_Projects\ts-loader\examples\vanilla\src\render.ts(3,34)
      TS2339: Property 'includes' does not exist on type '"foobar"'.
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

### Location of a Minimal Repository that Demonstrates the Issue.
I can provide a minimal repo in case the above steps are not sufficient.

**Comment by johnnyreilly** [comment]
I'm slightly surprised by this - we've a test pack that covers this case:

https://github.com/TypeStrong/ts-loader/tree/master/test/comparison-tests/es5

It could be worth comparing your setup with this. Either way, the fact that changing the version of TypeScript you use prompts the issue suggests it's not a ts-loader problem and actually a TypeScript one

**Comment by HermannGruber** [comment]
Thank you very much for the hint! So I'll try to understand the comparison-tests suite. In case I need to report the problem to the typescript folks: Ultimately ts-loader will call tsc. Is there a way to track down what this call is exactly (which arguments etc.)? I suspect that the typescript devs wouldn't be excited if I describe the issue by referring to ts-loader.

**Comment by johnnyreilly** [comment]
It's probably worth trying using tsc alone with the 2 different versions of TypeScript and seeing if the issue presents as a first step. ts-loader aims to be a drop in replacement for tsc behaviour wise (as a general rule)

**Comment by HermannGruber** [comment]
Thank you very much for the guidance. I tried tsc alone with the 2 different versions of TypeScript and could repro the (bad) behavior. I tried three further things:

I also tried to run the tests but they were not green on my machine (diffs in the output) when I ran them on master. I think I won't invest any further time into making them run on my machine at the moment.

The compiler errors appear (correct behavior) however once I copy the modified packages.config, tsconfig.json to another directory, followed by yarn install and tsc. 

Or if I modify the tsconfig.json in the original folder to include no types, the errors appear again (correct behavior):
````js
{
    "compilerOptions": {
        "sourceMap": true,
		"target": "es5", 
		"types": []
    }
}```

So it seems that the typescript compiler draws the prototype "include" is somehow from "@types" (I don't know exactly what that is at the moment).
